.gitignore

```diff
@@ -15,4 +15,4 @@ cache/
 artifacts/
 *flattened*
 *.gpg
-!private-key.gpg
+private-key*.gpg
```

Comment:
Also this would avoid us to check everytime a new encrypted private-key is pushed. I say it must be reviewed because somebody can mistakenly do something like this
```echo '<priv-key-content-here>' > private-key.gpg```

Reply:
While if there are no chances to push private key, we don't need to review them everytime

Reply:
Yeah, I see what you're saying, and in this case given it's private keys we should err on the side of caution. Case closed, we'll keep the ignore.